include/NovelRT/Graphics/RenderingService.h

```diff
@@ -53,6 +55,8 @@ namespace NovelRT::Graphics {
     void beginFrame() const;
     void endFrame() const;
 
+    void setBackgroundColour(const RGBAConfig colour);
```

Comment:
Don't we pass `RGBAConfig` by reference? Not that it matters too much I don't think in this case, but its marked `const` when you're passing by value, which is odd.

Reply:
Switched to pass by reference.